### PR TITLE
Add dedicated CSV source for Organisations

### PR DESCRIPTION
**Pull request by odlp**
Changes:

- Splits `data/inputs/apic.csv` into:
  - `data/organisation.csv` - containing the ID, Name, Alternative Name, and URL
  - `data/catalogue.csv` - largely the same as `apic.csv`, without the Organisation. Provider switches to contain the Organisation ID

### CODE REVIEW

File: data/operational.csv-metadata.json [hunk 1]
```diff
@@ -1,42 +0,0 @@
-{
```
Comment (arnau) [inline data/operational.csv-metadata.json:1]: Is this removal intentional?

File: data/organisation.csv [hunk 2]
```diff
@@ -0,0 +1,20 @@
+"id","name","alternateName","url"
```
Comment (arnau) [inline data/organisation.csv:1]: The organisation schema (deleted now) has a slightly different column name and order. Could you use `identifier` instead of `id` and reflect the change from `about` to `url` in the schema file?

**Review by arnau — CHANGES_REQUESTED** [review]

File: data/operational.csv-metadata.json (hunk 1 above)
Comment (odlp) [reply data/operational.csv-metadata.json:1]: Yes – my suggestion is to have two CSV sources. I think the third file to store date created & date modified introduces unnecessary complexity. 

Also, using the URL for the primary identifier doesn't work for all records - how do we provide dates for catalogue entries without URLs?

Comment (arnau) [reply data/operational.csv-metadata.json:1]: Agree that the URL as primary doesn't work at the moment. Options need to be considered though, e.g. requiring either the URL or the URL SHA256 if there is an issue with making it public. Having it optional is an option I'd like to avoid unless the previous idea is deemed unfeasible.

In regards to the split, then it means that the sources will be like they are now but the _published_ data will be different. I expect the CSVW to be one way to consume the catalogue.

Comment (arnau) [reply data/operational.csv-metadata.json:1]: Based on our conversation this morning, I think this is no longer relevant for this PR

**Review by arnau — APPROVED** [review]
